queries.sql

```diff
@@ -35,4 +35,43 @@ where name <> 'Gabumon';
 
 select *
 from animals
-where weight_kg between 10.5 and 17.3;
\ No newline at end of file
+where weight_kg between 10.5 and 17.3;
+
+
+-- set the species
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species from animals;
+ROLLBACK;
+SELECT species from animals;
+
+--  update the species
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT name, species from animals;
+
+-- delete the records
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] you should check how many lines there are in the animals table after deleting as well before the ROLLBACK